Swift Code Challenge/Views/UsersListView.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  UsersListView.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 29/06/22.
+//
+
+import SwiftUI
+
+struct UsersListView: View {
+    @ObservedObject var viewModel: UsersListViewModelAsync = UsersListViewModelAsync()
+    @State private var showDevice = false
+    @State private var selection: Person? = nil
+    
+    
+    var body: some View {
+        ZStack{
+            NavigationView{
+                if(viewModel.personList.isEmpty){
+                    ProgressView()
+                        .progressViewStyle(CircularProgressViewStyle())
+                }
+                else{
+                    VStack{
+                        ListComponent(viewModel: viewModel)
+                        
+                        NewUsersButton(viewModel: viewModel)
+                    }
+                    .toolbar {
+                        EditButton()
+                    }
+                }
+            } .navigationViewStyle(StackNavigationViewStyle())
```

Comment:
Specifying the navigation style isn't necessary unless there's a specific reason. 

```suggestion
            }
```